### ISSUE TITLE
Bug - Table - TableText tooltips not triggered via keyboard

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-react#8158]
When using keyboard to navigate tables, truncated text cannot receive keyboard focus. This results in the keyboard user being unable to trigger the tooltip to open and display the truncated text. In most cases, the truncated text is getting a tooltip dynamically by login in the `TableText` component.

Truncated `TableText` needs to be focusable when it is not in an interactable element. such as the truncated cells in the [controlling text example](https://www.patternfly.org/v4/components/table#composable-controlling-text).

In examples such as the [sortable and wrapping headers example](https://www.patternfly.org/v4/components/table#composable-controlling-text), the truncated `TableText` is actually placed within a button (used to control the sorting). In such cases, the tooltip likely needs to be built around the button, or attached to a different interactable element like a `(?)` icon, for example so that we avoid nesting interactable elements.

![Oct-04-2022 21-21-14](https://user-images.githubusercontent.com/26305082/193959169-c179b6ff-e125-457e-b2ae-c7e8fe54dbea.gif)

**Comment by evwilkin** [comment]
@Dominik-Petrik FYI any PR for this will need to target the `v5` branch rather than `main`
